### ISSUE TITLE
about the result of the xception2_net

### ISSUE THREAD

**Issue by bz2233** [codelearnern/tweakedSelfDrive#1]
hello,i have tried to train a network,but the result is not good.can you send me your xception2_net?Did that work out?

**Comment by ayaanp-dev** [comment]
Did you use the xception net i used in this repo?  Also, if you are using the xception on a different task than the self driving car, I would recommend using the pretrained xception net by tensorflow.  Here's a link to that https://www.tensorflow.org/api_docs/python/tf/keras/applications/Xception

This is the code for my xception, 

```python
def xception(width, height, lr):
    model = Sequential([
        Conv2D(32, 3, activation="relu", input_shape=(width, height, 1), padding="same"),
        Conv2D(64, 3, activation="relu", padding="same"),
        Conv2D(128, 3, padding="same"),
        Conv2D(128, 3, activation="relu", padding="same"),
        MaxPooling2D(3, 2),
        Conv2D(256, 3, activation="relu", padding="same"),
        Conv2D(256, 3, activation="relu", padding="same"),
        MaxPooling2D(3, 2),
        Conv2D(728, 3, activation="relu", padding="same"),
        Conv2D(728, 3, activation="relu", padding="same"),
        MaxPooling2D(3, 2),
        Conv2D(728, 3, activation="relu", padding="same"),
        Conv2D(728, 3, activation="relu", padding="same"),
        Conv2D(728, 3, activation="relu", padding="same"),
        Conv2D(728, 3, activation="relu", padding="same"),
        Conv2D(1024, 3, activation="relu", padding="same"),
        MaxPooling2D(3, 2),
        Conv2D(1536, 3, activation="relu", padding="same"),
        Conv2D(3, 3, activation="relu", padding="same"),
        GlobalAveragePooling2D(),
        ])
```

**Comment by bz2233** [comment]
yes,as you can see,this repo don't have the xception2_net right?I just run the train_model and get one xception2_net,but it seems to work really bad.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ayaan Panda"<notifications@github.com&gt;; 
发送时间: 2020年7月25日(星期六) 晚上10:56
收件人: "codelearnern/tweakedSelfDrive"<tweakedSelfDrive@noreply.github.com&gt;; 
抄送: "loneliness"<1749029425@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [codelearnern/tweakedSelfDrive] about the result of the xception2_net (#1)





 
Did you use the xception net i used in this repo?

**Comment by bz2233** [comment]
oh,i just seen this cause the e-mail only shows your first sentence.Sorry about that and thank you very much! i'm just a noob about programming. i know that you did not upload the xception2_net due to the file limit,so i run the train_model using your balanced trained data and get the xception2_net. it actually did not run completely finished because i trained this data in cpu and about 30 epochs the run auto shut down and the acc of train data( about 0.9222) the acc of validation(about 0.9000).the trained xception2_net seems only w and d.what about your xception2_net,did that worked very well?

**Comment by ayaanp-dev** [comment]
Same here!  The model gives a very good accuracy and works pretty well, it's just that for some reason it can only use the w and d keys.  I have checked the code and it seems right.  I'm gonna try to modify the code to let it use s and a keys.  Anyway, you're welcome!

**Comment by bz2233** [comment]
Oh,i got it.Thank you very much&nbsp; for your help and recommendation!Hope you can modify the model very well! i am going to following~




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "codelearnern/tweakedSelfDrive"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2020年7月25日(星期六) 晚上11:49
收件人:&nbsp;"codelearnern/tweakedSelfDrive"<tweakedSelfDrive@noreply.github.com&gt;;
抄送:&nbsp;"loneliness"<1749029425@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [codelearnern/tweakedSelfDrive] about the result of the xception2_net (#1)





 
Same here!  The model gives a very good accuracy and works pretty well, it's just that for some reason it can only use the w and d keys.  I have checked the code and it seems right.  I'm gonna try to modify the code to let it use s and a keys.  Anyway, you're welcome!

**Comment by ayaanp-dev** [comment]
No problem!
